account_payment_order/security/payment_security.xml

```diff
@@ -14,6 +14,10 @@
 <data noupdate="1">
 
 
+<record id="account.group_account_manager" model="res.groups">
```

Comment:
This one is not correct IMO and I have cases where account managers shouldn't be able to perform orders. Put that admin user belongs to this, but not the implied group.

Reply:
that's why it's noupdate and you configure your case the way you need it. As a default, I think this is very sensible.

Reply:
The same applies to you, so please leave it as it was.

Reply:
as written above, this is not for my private amusement, but has a technical reason:

> The implied group is necessary so that https://github.com/OCA/OCB/blob/12.0/addons/account/tests/account_test_users.py#L15 can create a bank. I think this makes sense anyways, account managers should be allowed to do anything concerning accounting by default.

I moved this to demo data then. But I still believe it's very undesirable for the general case that installing this module takes away permissions from account managers.

Reply:
For the technical reason, for me it's as simple as adding the group to the user in the setUp of your tests.

For the functional ones, I let other to give their opinion, but for me it's OK as it is.